cobalt/act.py

```diff
@@ -46,6 +51,14 @@ def __init__(self, xml=None):
     def to_xml(self):
         return etree.tostring(self.root, encoding='utf-8')
 
+    def get_namespace(self):
+        namespaces = sorted(list(AKN_NAMESPACES.values()), reverse=True)
+        possible_namespace = self.root.nsmap.get(None)
```

Comment:
This will only check the default (`None`) namespace -- in theory we support an AKN document where *any* namespace is the AKN namespace.